cve_bin_tool/cli.py

```diff
@@ -140,6 +140,19 @@ def main(argv=None):
         choices=["low", "medium", "high", "critical"],
         help="minimum CVE severity to report (default: low)",
     )
+    output_group.add_argument(
+        "-a",
+        "--append",
+        nargs="?",
+        const=True,
+        help="save output as intermediate report in json format",
+    )
+    output_group.add_argument(
+        "-t",
+        "--tag",
+        action="store",
+        help="save a tag in intermediate reports",
```

Comment:
make this help msg more specific we are not saving a tag in the report data. we are actually adding a sort of category to the report.